app/views/state_file/questions/nj_review/edit.html.erb

```diff
@@ -61,6 +61,22 @@
     </div>
   <% end %>
 
+    <% if !current_intake.claimed_as_eitc_qualifying_child_unfilled? || !current_intake.spouse_claimed_as_eitc_qualifying_child_unfilled? %>
+    <div id="eitc_qualifying_child" class="white-group">
+      <div class="spacing-below-5">
+        <% if !current_intake.claimed_as_eitc_qualifying_child_unfilled? %>
+          <p class="text--bold spacing-below-5"><%=t(".claimed_as_eitc_qualifying_child") %></p>
```

Comment:
[pebble] This needs to copy the h2 header pattern from the other review items. I'm seeing that a few other review items snuck in that are using styles instead of headings and conveying meaning only to sighted users-- we should throw in a fix for that or make a ticket

Reply:
good catch- yes i had copied from existing items so there are at least one to fix- i'll add to a followup ticket

Reply:
this one is fixed and  https://github.com/newjersey/affordability-pm/issues/153 links here